src/department.js

```diff
@@ -38,12 +38,17 @@ var Department = React.createClass({
         {...this.props}
         hideInput={true}
         hideResults={true} />
-      <div className="departmentPage">
-        <DepartmentDecorator key={deptName} department={deptName} params={this.props.params} departmentInfo={this.props.data.departments} />
-      </div>
-      <Peek facet="department" q={deptName} quiltProps={{maxRowHeight: 400}} offset={10}/>
-      <Helmet title={deptName} />
-      <Departments compact={true} active={deptName} />
+      <main role="main" className="departmentPage" id="main">
```

Comment:
By using the main element we help tell screenreaders/search engines where the primary content is. This can help consumers more quickly locate the unique content of this page rather than scaffolding around it (footer, navigation, etc).  

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main